NuKeeper.Inspection/RepositoryInspection/DirectoryBuildTargetsReader.cs

```diff
@@ -66,9 +70,18 @@ private PackageInProject XmlToPackage(XElement el, PackagePath path)
             {
                 id = el.Attribute("Update")?.Value;
             }
-            var version = el.Attribute("Version")?.Value ?? el.Element("Version")?.Value;
+            var version = el.Name == "PackageDownload"
+                ? GetVersion(el)?.Trim('[', ']')
+                : GetVersion(el);
 
             return _packageInProjectReader.Read(id, version, path, null);
         }
+
+        private static string GetVersion(XElement el, XNamespace ns = null)
```

Comment:
I wasn't sure if there is a good place to put common utilities like this, so I just used it twice, this can always be cleaned up later 😄